### PR TITLE
docs: add walkthrough for multi-connector setup

### PR DESCRIPTION
**Pull request by paullatzelsperger**
## WHAT

This PR adds a sample how to run a "mini dataspace" consisting of two connectors and a DAPS instance.

## WHY

Documentation

## FURTHER NOTES

.

Closes # <-- _insert Issue number if one exists_

### CODE REVIEW

**Comment by paullatzelsperger** [comment]
\cc @yurimssilva

File: docs/samples/example-dataspace/plato-values.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+
+install:
+  daps: false
+fullnameOverride: tx-plato
+################################
+# EDC ControlPlane + DataPlane #
+################################
+participant:
+  id: "test-participant"
```
Comment (wolf4ood) [inline docs/samples/example-dataspace/plato-values.yaml:28]:
```suggestion
  id: "BPNPLATO"
```

**Review by wolf4ood — CHANGES_REQUESTED** [review]
Tested locally and worked fine, just small change on participant id

**Review by wolf4ood — APPROVED** [review]
:rocket:

**Pull request merged**